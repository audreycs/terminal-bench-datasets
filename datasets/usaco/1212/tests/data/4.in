1000 2
1339 1300 1390 1429
1720 1651 1740 1808
395 425 515 485
938 970 1061 1029
252 209 298 344
589 595 685 677
1113 1163 1254 1203
1547 1460 1550 1637
618 567 657 707
274 200 302 315
958 947 1041 1049
939 969 1060 1030
1137 1139 1230 1227
1488 1519 1610 1577
1108 1169 1259 1197
758 787 878 845
1906 1823 1914 1996
1114 1162 1253 1204
1691 1679 1769 1780
1651 1717 1808 1741
1154 1121 1213 1246
558 624 715 649
1297 1342 1431 1387
617 568 658 706
1675 1693 1785 1766
565 615 707 656
976 930 1022 1065
5 88 178 95
813 730 820 903
561 621 711 652
1130 1146 1236 1220
1170 1106 1198 1262
1175 1101 1192 1266
1668 1701 1792 1758
1880 1851 1943 1968
735 808 898 825
1478 1528 1619 1568
1694 1676 1765 1783
1692 1678 1768 1781
27 65 157 117
559 623 713 650
996 911 1002 1086
772 773 863 860
796 745 837 886
1536 1471 1561 1626
937 939 1000 1021
1538 1469 1559 1628
446 377 467 537
601 584 674 690
1337 1302 1392 1427
746 797 887 836
367 456 546 457
1348 1291 1381 1438
604 581 670 693
203 258 348 296
1458 1458 1639 1639
1141 1134 1225 1233
1126 1150 1240 1216
1526 1481 1572 1616
1652 1716 1807 1742
1871 1862 1952 1959
926 980 1071 1018
933 960 1029 1066
1319 1320 1410 1408
1667 1702 1793 1757
190 271 361 281
1346 1293 1383 1436
588 596 686 676
1676 1692 1784 1767
1299 1340 1429 1389
1469 1537 1628 1559
42 50 142 132
1697 1673 1761 1786
1717 1654 1743 1805
1724 1646 1736 1813
1868 1865 1955 1956
62 31 123 151
794 747 839 884
1686 1669 1777 1810
1467 1539 1630 1557
1482 1524 1615 1572
1857 1876 1967 1946
246 215 305 338
1342 1297 1387 1432
1678 1690 1782 1769
975 931 1023 1064
1158 1117 1208 1250
1664 1704 1795 1755
16 76 168 106
1826 1905 1995 1914
1679 1689 1780 1770
81 12 102 170
1721 1649 1739 1809
1306 1332 1422 1396
1671 1698 1789 1762
1358 1281 1371 1448
387 433 524 478
380 441 531 469
43 49 141 133
553 629 720 643
911 996 1086 1001
1302 1336 1426 1392
1292 1347 1436 1382
1901 1828 1919 1991
577 604 696 667
949 957 1050 1040
187 274 364 277
814 729 819 904
444 379 471 535
1529 1478 1569 1619
1873 1859 1950 1961
1860 1872 1963 1949
189 272 362 280
1859 1873 1964 1948
567 613 705 658
183 183 1090 1090
63 30 122 152
1132 1144 1234 1223
88 5 95 178
1127 1149 1239 1217
398 422 512 489
1540 1467 1557 1630
925 981 1072 1017
1869 1864 1954 1957
944 964 1055 1035
741 802 892 831
441 381 473 533
394 426 516 484
36 56 148 126
1711 1643 1781 1816
978 928 1020 1069
52 40 132 142
930 976 1067 1023
1515 1492 1583 1605
1468 1538 1629 1558
1881 1850 1942 1969
1460 1546 1637 1550
633 550 641 723
248 213 303 340
1275 1364 1453 1365
392 427 517 483
206 255 345 300
64 29 121 153
628 556 646 718
1897 1832 1924 1987
430 391 483 522
1341 1298 1388 1431
1503 1504 1596 1593
1712 1658 1747 1801
1874 1858 1949 1962
1827 1904 1994 1915
1648 1721 1811 1738
784 758 849 872
210 251 341 304
443 380 472 534
393 447 520 473
568 555 714 668
1321 1318 1408 1410
74 19 110 163
1834 1898 1988 1924
432 455 460 528
1148 1128 1219 1240
572 609 701 662
1681 1688 1779 1771
1713 1657 1746 1802
775 770 859 863
425 396 488 515
1143 1132 1223 1235
1466 1540 1631 1556
1541 1466 1556 1631
1640 1729 1819 1730
244 217 307 336
774 771 860 862
18 74 166 108
1475 1531 1622 1565
450 374 464 540
762 783 873 849
1835 1897 1987 1925
742 801 891 832
1714 1656 1745 1803
86 7 97 176
1336 1303 1393 1426
1351 1288 1378 1441
1863 1869 1960 1952
445 378 468 536
1474 1532 1623 1564
743 800 890 833
1725 1645 1735 1814
1480 1526 1617 1570
77 16 107 166
410 409 501 503
740 803 893 830
1111 1166 1256 1201
232 232 320 324
1508 1499 1590 1598
919 988 1078 1011
218 244 333 311
757 788 879 844
1539 1468 1558 1629
964 942 1035 1054
1293 1346 1435 1383
620 564 655 710
1673 1696 1787 1764
1654 1714 1805 1744
1101 1175 1266 1192
961 945 1038 1051
1866 1866 1957 1955
1459 1548 1638 1549
786 756 847 874
369 453 544 459
1535 1472 1562 1625
730 759 829 887
242 220 310 333
1120 1156 1246 1211
1320 1319 1409 1409
1703 1666 1755 1793
186 275 365 276
48 44 136 138
755 789 880 843
55 38 130 144
1870 1863 1953 1958
954 952 1045 1045
1281 1358 1447 1371
629 554 645 719
927 979 1070 1019
584 570 693 699
1701 1668 1757 1790
423 398 490 513
1690 1680 1770 1779
582 601 692 671
1514 1493 1584 1604
197 264 354 290
1328 1311 1401 1418
1710 1659 1748 1800
230 233 321 323
805 738 830 895
931 975 1066 1024
970 936 1028 1059
1094 1183 1273 1185
30 62 154 120
411 401 538 495
1716 1655 1744 1804
1462 1544 1635 1552
551 631 722 641
982 924 1016 1073
1360 1279 1369 1450
427 394 486 517
1153 1122 1214 1245
1301 1337 1427 1391
779 764 854 867
560 622 712 651
1177 1100 1191 1267
1279 1360 1449 1369
787 755 846 875
1864 1868 1959 1953
936 971 1062 1028
1303 1335 1425 1393
223 239 327 317
1481 1525 1616 1571
1289 1350 1439 1379
371 451 542 461
1647 1722 1812 1737
71 22 113 160
1350 1289 1379 1440
1509 1498 1589 1599
1494 1514 1605 1583
1852 1881 1972 1939
920 987 1077 1012
922 984 1075 1014
1151 1124 1216 1243
1704 1664 1754 1794
748 794 885 838
1179 1098 1189 1269
608 577 666 697
24 68 160 114
792 749 841 882
1685 1684 1774 1775
404 414 506 497
566 614 706 657
780 763 853 868
1330 1309 1399 1420
924 982 1073 1016
7 87 177 96
184 184 907 907
26 66 158 116
1099 1177 1268 1190
745 798 888 835
1123 1136 1229 1222
37 55 147 127
260 199 290 353
750 793 884 839
1162 1114 1205 1254
1707 1662 1752 1796
1845 1888 1977 1935
1300 1338 1428 1390
51 41 133 141
1134 1143 1233 1224
215 246 335 309
216 245 334 310
1849 1886 1974 1937
412 408 499 504
1121 1155 1245 1212
621 563 654 711
595 589 679 684
1278 1361 1450 1368
1345 1294 1384 1435
422 399 491 512
402 416 508 494
778 765 855 866
1461 1545 1636 1551
418 403 494 509
1836 1896 1986 1926
1174 1102 1193 1265
413 407 498 505
1702 1667 1756 1791
1879 1852 1944 1967
1516 1491 1582 1606
929 977 1068 1022
268 191 283 360
400 418 510 492
76 17 108 165
605 580 669 694
1324 1315 1405 1414
1150 1125 1217 1242
1139 1137 1227 1230
1672 1697 1788 1763
1533 1474 1564 1623
1098 1178 1269 1189
812 731 821 902
1655 1713 1804 1745
417 404 495 508
1283 1356 1445 1373
3 90 180 93
1531 1476 1567 1621
771 774 864 859
390 429 519 481
943 965 1056 1034
1900 1829 1920 1990
211 250 340 305
415 406 497 506
769 776 866 857
1718 1653 1742 1806
798 744 836 888
1828 1903 1993 1916
585 599 690 673
1116 1160 1251 1207
257 204 293 350
611 575 664 700
1824 1907 1997 1912
810 733 823 900
783 760 850 871
1832 1900 1990 1920
749 795 877 853
1167 1108 1200 1260
91 2 92 181
1642 1727 1817 1732
1340 1299 1389 1430
40 52 144 130
47 45 137 137
1519 1488 1579 1609
1472 1534 1625 1562
266 193 284 359
1135 1142 1232 1225
66 27 119 155
1855 1878 1969 1942
969 937 1031 1058
28 64 156 118
777 768 857 865
1837 1895 1985 1927
1159 1116 1207 1251
729 814 903 819
434 388 480 525
1666 1703 1794 1756
1890 1841 1931 1979
1661 1707 1798 1751
78 15 106 167
1684 1685 1775 1774
563 617 709 654
1665 1694 1763 1753
1853 1880 1971 1940
370 452 543 460
776 769 858 864
785 757 848 873
1352 1287 1377 1442
191 270 360 283
1659 1709 1800 1749
1479 1527 1618 1569
1843 1890 1980 1933
981 925 1017 1072
557 625 716 647
737 806 896 827
799 743 835 889
731 813 902 821
988 919 1011 1078
1705 1718 1734 1798
228 234 322 322
759 786 876 846
61 32 124 150
1314 1325 1415 1403
1463 1543 1634 1553
1505 1502 1594 1595
1892 1838 1929 1981
578 603 695 669
1886 1845 1936 1975
1169 1107 1199 1261
736 807 897 826
72 21 112 161
41 51 143 131
1483 1523 1614 1573
989 918 1010 1079
569 612 704 659
1682 1687 1778 1772
1361 1278 1368 1451
379 442 532 468
1820 1909 1999 1910
1316 1323 1413 1405
1347 1292 1382 1437
243 218 309 334
1093 1093 1274 1274
1524 1483 1574 1614
1865 1867 1958 1954
1355 1284 1374 1445
1313 1326 1416 1402
68 25 117 157
49 43 135 139
10 82 174 99
979 927 1019 1070
1883 1848 1940 1972
1106 1126 1197 1206
416 405 496 507
406 412 504 499
1161 1173 1243 1228
193 268 358 285
627 557 647 717
1848 1883 1937 1945
1830 1901 1991 1918
552 630 721 642
1831 1874 1956 1921
255 206 295 348
1674 1695 1786 1765
1335 1304 1394 1425
33 59 151 123
797 766 906 820
934 973 1064 1026
948 958 1051 1039
70 23 114 159
977 929 1021 1068
587 597 687 675
217 192 281 335
1311 1328 1418 1400
1317 1322 1412 1406
609 576 665 698
240 222 312 331
556 626 717 646
1722 1648 1738 1811
1646 1723 1813 1736
1726 1644 1733 1815
1907 1822 1913 1997
997 910 1001 1087
1288 1351 1440 1378
957 948 1042 1048
923 983 1074 1015
602 583 673 691
8 86 176 97
1543 1464 1554 1633
1318 1321 1411 1407
386 434 525 477
259 201 291 352
426 395 487 516
1527 1480 1571 1617
548 635 725 638
1312 1327 1417 1401
1296 1343 1432 1386
1096 1181 1271 1187
1521 1486 1577 1611
388 432 522 479
744 799 889 834
1362 1277 1367 1452
1723 1647 1737 1812
747 796 886 837
1885 1846 1938 1974
1833 1899 1989 1923
1305 1333 1423 1395
1905 1833 1932 1985
963 943 1036 1053
29 63 155 119
1291 1348 1437 1381
984 923 1015 1074
793 748 840 883
436 386 478 527
80 13 103 169
1856 1877 1968 1944
1276 1363 1452 1366
263 196 287 356
1152 1123 1215 1244
1887 1844 1935 1976
781 762 852 869
626 558 648 716
591 593 683 679
85 8 98 175
945 963 1054 1036
1867 1826 1979 1919
1903 1825 1917 1993
1489 1547 1566 1586
53 84 116 173
1862 1870 1961 1951
1497 1511 1602 1587
1327 1312 1402 1417
619 566 656 708
1660 1708 1799 1750
270 189 279 362
1645 1724 1814 1735
224 238 326 318
241 221 311 332
1315 1324 1414 1404
271 188 278 363
220 242 331 313
1117 1159 1249 1208
205 256 346 298
1512 1495 1586 1602
767 778 868 855
1491 1517 1608 1579
912 995 1085 1003
397 423 513 488
226 236 324 320
419 402 493 510
782 761 851 870
727 816 905 817
1504 1503 1595 1594
1131 1145 1235 1221
23 69 161 113
372 450 540 462
1138 1138 1228 1229
1644 1725 1815 1734
1838 1894 1984 1928
562 620 710 653
1119 1157 1247 1210
1334 1305 1395 1424
231 231 318 278
1662 1706 1797 1752
235 227 316 327
1522 1485 1576 1612
1349 1290 1380 1439
1294 1345 1434 1384
87 6 96 177
1285 1354 1443 1375
1105 1140 1250 1239
807 736 826 897
1333 1306 1396 1423
396 424 514 486
728 815 904 818
1643 1726 1816 1733
806 737 828 896
816 727 817 906
962 944 1037 1052
576 605 697 666
73 20 111 162
267 202 328 282
579 619 653 648
25 67 159 115
1657 1711 1802 1747
592 592 682 680
610 618 672 689
1168 1104 1260 1198
65 28 120 154
754 750 856 876
258 203 292 351
1693 1677 1767 1782
614 572 661 703
1149 1127 1218 1241
212 249 339 306
554 628 719 644
1323 1316 1406 1413
603 582 671 692
399 419 511 490
791 751 842 880
1343 1296 1386 1433
75 18 109 164
1097 1180 1270 1188
974 932 1024 1063
414 420 470 491
229 228 308 346
607 578 667 696
1677 1691 1783 1768
448 430 500 519
941 967 1058 1032
1523 1484 1575 1613
1877 1855 1946 1965
1477 1529 1620 1567
987 920 1012 1077
1498 1510 1601 1588
1727 1642 1732 1817
1147 1129 1220 1238
1344 1295 1385 1434
732 812 901 822
1696 1674 1762 1785
455 368 458 545
1359 1280 1370 1449
573 608 700 663
17 75 167 107
1499 1509 1600 1589
915 992 1082 1007
995 912 1003 1085
631 552 643 721
916 991 1081 1008
615 571 660 704
1363 1276 1366 1453
1104 1171 1262 1195
606 579 668 695
1500 1508 1599 1590
227 235 323 321
1304 1334 1424 1394
213 248 337 307
439 383 475 531
50 42 134 140
378 445 536 487
265 194 285 358
385 435 526 475
1145 1130 1221 1237
1546 1461 1551 1636
950 956 1049 1041
1687 1683 1773 1776
1322 1317 1407 1411
208 253 343 302
803 740 832 893
249 212 301 341
1510 1497 1588 1600
1822 1854 1915 1922
421 400 492 511
214 247 336 308
1501 1507 1598 1591
1183 1094 1184 1273
1688 1682 1772 1777
788 754 845 877
1129 1147 1237 1219
1485 1521 1612 1575
1663 1705 1796 1754
22 70 162 112
994 913 1004 1084
1854 1879 1970 1941
440 382 474 532
233 230 319 325
956 949 1043 1047
1658 1710 1801 1748
89 4 94 179
1904 1824 1916 1995
1902 1827 1918 1992
453 371 461 543
751 792 883 840
808 735 825 898
1173 1103 1194 1264
547 636 726 637
1708 1661 1751 1797
38 54 146 128
942 966 1057 1033
451 373 463 541
932 974 1065 1025
200 261 351 293
250 211 300 342
795 746 838 885
1908 1821 1911 1998
804 739 831 894
1180 1097 1188 1270
1656 1712 1803 1746
1825 1906 1996 1913
581 602 694 670
435 387 479 526
35 57 149 125
564 616 708 655
951 955 1048 1042
1 1 182 182
1309 1339 1454 1412
1492 1516 1607 1580
910 997 1087 1000
1899 1830 1922 1989
1669 1700 1791 1760
196 265 355 289
947 961 1052 1038
636 547 637 726
408 411 503 500
921 986 1076 1013
1909 1820 1910 1999
1109 1168 1258 1199
1103 1172 1263 1194
1326 1313 1403 1416
985 922 1014 1075
407 421 523 520
800 767 861 892
1502 1505 1597 1592
1484 1522 1613 1574
21 71 163 111
1858 1875 1965 1947
1473 1533 1624 1563
761 784 874 848
765 780 870 852
992 915 1007 1082
738 805 895 828
219 243 332 312
204 257 347 297
447 376 466 538
456 367 457 546
376 444 534 466
1122 1154 1244 1213
401 417 509 493
1534 1473 1563 1624
1338 1301 1391 1428
1156 1119 1210 1248
1545 1462 1552 1635
1700 1670 1758 1789
1144 1131 1222 1236
1353 1286 1376 1443
917 990 1080 1009
952 954 1047 1043
84 9 99 174
11 81 173 100
1698 1672 1760 1787
1653 1715 1806 1743
1471 1535 1626 1561
374 448 537 464
1542 1465 1555 1632
452 372 462 542
1107 1170 1261 1196
811 732 822 901
1884 1847 1939 1973
1146 1165 1195 1252
437 385 477 529
201 260 350 294
1823 1908 1998 1911
1490 1518 1609 1578
1889 1842 1933 1978
381 440 530 470
623 561 651 713
1680 1650 1750 1759
79 14 104 168
1290 1349 1438 1380
1894 1836 1927 1983
454 369 459 544
83 10 100 172
622 562 652 712
1532 1475 1565 1622
251 210 299 343
971 935 1027 1060
373 449 539 463
1719 1652 1741 1807
1842 1891 1981 1932
625 559 649 715
234 229 317 326
273 186 276 365
58 35 127 147
790 752 843 879
1683 1686 1776 1773
254 207 296 347
1525 1482 1573 1615
269 190 280 361
1136 1141 1231 1226
256 205 294 349
763 782 872 850
733 811 900 823
420 438 469 501
594 590 680 682
262 197 288 355
953 953 1046 1044
1821 1840 1975 1994
272 187 277 364
1649 1720 1810 1739
19 73 165 109
239 223 313 330
908 908 1089 1089
1893 1837 1928 1982
1310 1329 1419 1399
1091 1091 1456 1456
1110 1167 1257 1200
194 267 357 286
913 994 1084 1005
389 431 521 480
1163 1113 1204 1255
13 79 171 102
46 46 138 136
1689 1681 1771 1778
56 37 129 145
773 772 862 861
990 917 1009 1080
1695 1675 1764 1784
1882 1849 1941 1970
1875 1857 1948 1963
815 728 818 905
940 968 1059 1031
598 587 677 686
405 413 505 498
596 565 688 709
764 781 871 851
1181 1096 1186 1271
1506 1501 1593 1596
1895 1835 1926 1984
739 804 894 829
236 226 315 328
20 72 164 110
1280 1359 1448 1370
247 214 304 339
1178 1099 1190 1268
613 573 662 702
928 978 1069 1020
188 273 363 279
1095 1182 1272 1186
225 237 325 319
1641 1728 1818 1731
580 634 638 683
1528 1479 1570 1618
753 790 881 842
1325 1314 1404 1415
60 33 125 149
1486 1506 1591 1582
760 785 875 847
1115 1161 1252 1205
914 993 1083 1006
1670 1699 1790 1761
1465 1541 1632 1555
57 36 128 146
238 224 314 329
39 53 145 129
1495 1513 1604 1584
15 77 169 105
429 392 484 521
1157 1118 1209 1249
1839 1885 1912 1931
1530 1477 1568 1620
599 586 676 687
261 198 289 354
1160 1115 1206 1253
375 446 535 465
424 397 489 514
1840 1893 1983 1929
600 585 675 688
1125 1152 1241 1215
1155 1120 1211 1247
1496 1512 1603 1585
1841 1892 1982 1930
991 916 1008 1081
1850 1882 1921 1971
1513 1494 1585 1603
549 633 724 639
192 269 359 284
1896 1834 1925 1986
199 262 352 292
31 61 153 121
1164 1112 1203 1256
1729 1640 1730 1819
909 998 1088 999
1544 1463 1553 1634
6 83 105 104
555 627 718 645
1287 1352 1441 1377
1493 1515 1606 1581
45 47 139 135
1286 1353 1442 1376
59 34 126 148
550 632 723 640
960 946 1039 1050
4 89 179 94
955 950 1044 1046
1898 1831 1923 1988
1092 1092 1455 1455
590 594 684 678
770 775 865 858
1357 1282 1372 1447
1706 1663 1753 1795
69 24 115 158
403 415 507 496
967 938 1032 1057
1709 1660 1749 1799
752 791 882 841
9 85 175 98
195 266 356 288
1872 1860 1951 1960
993 914 1006 1083
766 779 869 854
1699 1671 1759 1788
1142 1133 1224 1234
1354 1285 1375 1444
2 91 181 92
1331 1308 1398 1421
1715 1665 1766 1792
1133 1179 1265 1231
1277 1362 1451 1367
1851 1884 1973 1938
593 591 681 681
1124 1153 1242 1214
1507 1500 1592 1597
449 375 465 539
1176 1109 1187 1257
1470 1536 1627 1560
1140 1135 1226 1232
1102 1174 1264 1193
209 252 342 303
768 777 867 856
634 549 640 724
67 26 118 156
756 810 827 881
575 606 698 665
980 926 1018 1071
1307 1331 1421 1397
1100 1176 1267 1191
409 410 502 502
1118 1158 1248 1209
1728 1641 1731 1818
1298 1341 1430 1388
1282 1357 1446 1372
973 933 1025 1062
368 454 545 458
264 195 286 357
1457 1457 2000 2000
1182 1095 1185 1272
1846 1887 1976 1936
253 208 297 345
1829 1902 1992 1917
14 78 170 103
571 610 702 661
431 390 482 523
44 48 140 134
574 607 699 664
1112 1164 1255 1202
438 384 476 530
383 437 528 472
428 393 485 518
624 560 650 714
630 553 644 720
1520 1487 1578 1610
82 11 101 171
1844 1889 1978 1934
1308 1330 1420 1398
384 436 527 474
1847 1861 1966 1943
1172 1151 1212 1184
1356 1283 1373 1446
54 39 131 143
986 921 1013 1076
1166 1110 1201 1259
597 588 678 685
1518 1489 1580 1608
185 185 366 366
442 370 541 476
34 58 150 124
583 600 691 672
1284 1355 1444 1374
809 734 824 899
90 3 93 180
1476 1530 1621 1566
1891 1839 1930 1980
377 443 533 467
1464 1542 1633 1554
789 753 844 878
207 254 344 301
1876 1856 1947 1964
1650 1719 1809 1740
802 741 833 891
1295 1344 1433 1385
616 569 659 705
382 439 529 471
983 951 1040 1002
946 962 1053 1037
966 940 1033 1056
918 989 1079 1010
222 240 329 316
1511 1496 1587 1601
202 259 349 295
632 551 642 722
1128 1148 1238 1218
972 934 1026 1061
1165 1111 1202 1258
245 216 306 337
1329 1310 1400 1419
1517 1490 1581 1607
1888 1843 1934 1977
570 611 703 660
1332 1307 1397 1422
635 548 639 725
586 598 689 674
801 742 834 890
221 241 330 314
734 809 899 824
1537 1470 1560 1627
391 428 518 482
12 80 172 101
998 909 999 1088
965 941 1034 1055
275 225 338 287
1171 1105 1196 1263
935 972 1063 1027
612 574 663 701
32 60 152 122
959 985 1005 1067
433 389 481 524
1364 1275 1365 1454
1487 1520 1611 1576
968 959 1030 1004
1861 1871 1962 1950
1878 1853 1945 1966
237 219 282 299
198 263 353 291
1548 1459 1549 1638
