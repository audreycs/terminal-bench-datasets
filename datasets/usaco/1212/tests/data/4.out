84631 84729
